packages/design-systems/src/components/composite/Datagrid/SearchFilter/useCustomFilter.ts

```diff
@@ -223,29 +255,12 @@ export const useCustomFilter = <TData>({
     [],
   );
 
-  const numberOfSearchConditions = useMemo(() => {
-    const isCurrentStateValid = (state: FilterRowData) => {
-      return (
-        state.currentState.column &&
-        state.currentState.condition &&
-        state.currentState.value
-      );
-    };
-    const count = filterRows.reduce((acc, row, index) => {
-      // First row does not have jointCondition
-      if (index === 0) {
-        if (isCurrentStateValid(row)) {
-          return acc + 1;
-        }
-        return acc;
-      }
-      if (isCurrentStateValid(row) && row.currentState.jointCondition) {
-        return acc + 1;
-      }
-      return acc;
-    }, 0);
-    return count;
-  }, [filterRows]);
+  const changePrevFilterRows = useCallback(() => {
```

Comment:
外部から変更(change)するための引数を受け取らず、どちらかというとリセットする処理だと思うので以下の命名でお願いします。
```suggestion
  const resetToPrevFilterRows = useCallback(() => {
```